src/cli/pack_command.zig

```diff
@@ -2455,22 +2455,15 @@ pub const PackCommand = struct {
             isSpecialFileOrVariant(filename, "README"));
     }
 
-    // TODO: should this be case insensitive on all platforms?
-    const stringsEql = if (Environment.isLinux)
-        strings.eqlComptime
-    else
-        strings.eqlCaseInsensitiveASCIIICheckLength;
+    const stringsEql = strings.eqlCaseInsensitiveASCIIICheckLength;
```

Comment:
```suggestion
```

let's just delete this and replace it's one usage

Reply:
its just... such a long name